fast64_internal/oot/actor/properties.py

```diff
@@ -248,6 +251,17 @@ def draw_props(self, layout: UILayout, obj: Object, altSceneProp: OOTAlternateSc
 )
 
 
+@persistent
+def actorHandler(_: Scene):
+    for parentObj in bpy.data.objects:
```

Comment:
You should check if the game editor mode is "OOT" before executing the rest of this code.